clients/src/test/resources/common/message/StructMessage.json

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 {
-  "name": "NullableStructMessage",
+  "name": "StructMessage",
```

Comment:
Is this change needed? We do have other messages which contains non nullable structs. Could we extend those?

Reply:
Now that I've address the other review comment, the rename no longer makes sense. 

`SimpleExampleMessageTest` and `NullableMessageTest` exercise all combinations of {non-tagged, tagged}x{non-nullable, nullable} structs, so adding a size assert to both those classes is simpler.
That just leaves the particular 127-long tagged nullable struct edge case. I've left the test for that in `NullableMessageTest`.